tests/bson/bson-fromPHP_error-008.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+BSON\fromPHP(): PHP documents with circular references
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+echo "\nTesting packed array with circular reference\n";
+
+echo throws(function() {
+    $document = ['x' => 1, 'y' => [1, 2, 3]];
+    $document['y'][] = &$document['y'];
+    fromPHP($document);
+}, 'MongoDB\Driver\Exception\UnexpectedValueException'), "\n";
+
+echo "\nTesting associative array with circular reference\n";
+
+echo throws(function() {
+    $document = ['x' => 1, 'y' => []];
+    $document['y'][0]['z'] = &$document['y'];
+    fromPHP($document);
+}, 'MongoDB\Driver\Exception\UnexpectedValueException'), "\n";
+
+echo "\nTesting object with circular reference\n";
+
+echo throws(function() {
+    $document = (object) ['x' => 1, 'y' => (object) []];
+    $document->y->z = &$document->y;
+    fromPHP($document);
+}, 'MongoDB\Driver\Exception\UnexpectedValueException'), "\n";
```

Comment:
This test case is identical to the one in `bson-fromPHP_error-004.phpt` and should be removed.